docs/source/docker/getting_started/hardware_recommendations.rst

```diff
@@ -0,0 +1,88 @@
+.. _ref-hardware-recommendations:
+
+Hardware recommendations
+========================
+
+Lightly worker is usually run on dedicated hardware
+or in the cloud on a compute instance
+which is specifically spun up to run Lightly Worker standalone.
+Our recommendations on the hardware requirements of this compute instance are
+based on three criteria:
+
+- speed: The worker should process your dataset as quickly a possible.
+- cost-effectiveness: The compute instance should be economical.
+- stability: The worker should not crash because it runs out of memory.
+
+Depending on your dataset size, we recommend the following machine:
+
+- Up to 100.000 images or video frames: Use the AWS EC2 instance `g4dn.xlarge` or similar
+  with 4 vCPUs, 16GB of system memory, one T4 GPU
+- Up to 1 Million images or video frames: Use the AWS EC2 instance `g4dn.2xlarge` or similar
+  with 8 vCPUs, 32GB of system memory, one T4 GPU
+- More than 1 Million images or video frames: Use the AWS EC2 instance `g4dn.4xlarge` or similar
+  with 16 vCPUs, 64GB of system memory, one T4 GPU
+
+You can compute the number of frames of your videos with their length and fps.
+E.g. 100 videos with 600s length each and 30 fps have 100 * 600 * 30 = 1.8 Mio frames.
+
+If you want to train an embedding model for many epochs or want to further increase computing speed,
+we recommend to switch to a V100 or A10 GPU or better.
+
+If you read the data from a local hard disk, make sure that you use a SSD.
```

Comment:
Maybe we put this in brackets or add that this is not recommended. E.g. we recommend to use the workflow where data sits in a cloud bucket

Reply:
isnt the IO also limited on a cloud instance if you dont use a SSD? 🤔

Reply:
With local I meant the old workflow of reading from an input dir, which can also be used with a cloud instance. When streaming from a datasource, however, I/O to the local SSD or HDD should not matter, as the data flows directly from S3 bucket into RAM.

Reply:
I changed the order of the sentences and made clear that we recommend the workflow to stream from a cloud bucket.